package.json

```diff
@@ -36,14 +36,19 @@
     "react-router-dom": ">=5.1.2"
   },
   "devDependencies": {
-    "@babel/core": "^7.0.0-0",
+    "@babel/core": "^7.18.10",
     "@lifeomic/chromicons": "^2.2.0",
-    "@storybook/addon-a11y": "^5.3.18",
-    "@storybook/addon-actions": "^5.3.18",
+    "@storybook/addon-a11y": "^6.5.10",
+    "@storybook/addon-actions": "^6.5.10",
+    "@storybook/addon-essentials": "^6.5.10",
+    "@storybook/addon-interactions": "^6.5.10",
     "@storybook/addon-knobs": "^5.3.18",
```

Comment:
If we're not using knobs anymore, should this import be removed?

Reply:
Good call, I'm still running the storybook with the old components included, will remove at the end